xla/service/gpu/runtime/annotation.cc

```diff
@@ -491,8 +484,8 @@ ModuleAnnotations::ModuleAnnotations(std::string_view module_name)
 uint64_t KernelAnnotation::NvtxSchemaId() {
   static std::uint64_t schema_id = []() -> std::uint64_t {
 #if GOOGLE_CUDA
```

Comment:
not needed?

Reply:
https://github.com/openxla/xla/pull/9896#discussion_r1501037799